### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by mishakoshilka**
- [DEMO LINK](https://mishakoshilka.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://mishakoshilka.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,16 +1,57 @@
...
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <!-- Big box -->
+    <form
+    action="#"
+    method="get"
+    class="box big"
+    data-qa="big">
```
Comment (volodymyr-soltys97) [inline src/index.html:22]: You need to check and fix it everywhere
image

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's fix the problem with tests. You need to check styles for `font-family`

File: src/index.html [hunk 2]
```diff
@@ -1,16 +1,63 @@
...
+      method="get"
+      class="box big"
+      data-qa="big"
+    >
+      <img
+        src="images/Search.svg"
+        alt="search logo"
+        class="big-logo"
+      >
+
```
Comment (Viktor-Kost) [inline src/index.html:29]: an extra empty line 
```suggestion
```

File: src/index.html [hunk 3]
```diff
@@ -1,16 +1,63 @@
...
+      method="get"
+      class="box small"
+      data-qa="small"
+    >
+      <img
+        src="images/Search.svg"
+        alt="search logo"
+        class="small-logo"
+      >
+
```
Comment (Viktor-Kost) [inline src/index.html:52]: an extra empty line
```suggestion
```

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+}
+
+/* #region big */
```
Comment (Viktor-Kost) [inline src/style.css:34]: no needs to add such comments
```suggestion
```

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+  color: #687c94;
+  margin-left: 25px;
+  margin-bottom: 2px;
+}
+
+.search {
+  font-family: "Avenir", sans-serif;
+  flex: 1;
+  font-style: normal;
+  line-height: normal;
```
Comment (Viktor-Kost) [inline src/style.css:49]: use numeric value

Example:
line-height: 16px;

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+.form__input:focus {
+  text-shadow: 1px 4px 4px rgba(0, 0, 0, 0.25);
+  font-weight: 700;
+}
+
+.form__input::placeholder {
+  font-family: "Avenir-Book", sans-serif;
+  color: #687c94;
+  font-style: normal;
```
Comment (volodymyr-soltys97) [inline src/style.css:75]: No need to write `normal`. This is default value
```suggestion
```

**Review by volodymyr-soltys97 — APPROVED** [review]

**Pull request closed without merging**